src/app/components/statistics/pages/statistics.component.ts

```diff
@@ -1,128 +1,123 @@
-        value: "Unknown",
-      },
-      {
-        icon: defaultAudioIcon,
-        name: "Audio recordings",
-        value: "Unknown",
-      },
-      {
-        icon: defaultUserIcon,
-        name: "Users",
-        value: "Unknown",
-      },
-    ]);
+    const updateValues = (
+      group: "groupOne" | "groupTwo",
+      ...values: { index: number; value: string | number }[]
+    ) => {
+      let list: List<IItem> = this[group];
+      values.forEach(({ value, index }) => {
+        list = list.set(index, { ...list.get(index), value });
+      });
+      this[group] = list;
+    };
 
-    this.groupTwo = List([
-      {
-        icon: ["fas", "tags"],
-        name: "Unique tags attached to annotations",
-        value: "Unknown",
-      },
-      {
-        icon: ["fas", "tags"],
-        name: "Tags attached to annotations",
-        value: "Unknown",
-      },
-      {
-        icon: ["fas", "bullseye"],
-        name: "New annotations in the last month",
-        value: "Unknown",
-      },
-      {
-        icon: ["fas", "clock"],
-        name: "Overall annotation duration",
-        value: "Unknown",
-      },
-      {
-        icon: defaultUserIcon,
-        name: "Users Online",
-        value: "Unknown",
-      },
-      {
-        icon: defaultAudioIcon,
-        name: "Overall audio recording file size",
-        value: "Unknown",
-      },
-      {
-        icon: defaultAudioIcon,
-        name: "New audio recordings in last month",
-        value: "Unknown",
-      },
-      {
-        icon: ["fas", "clock"],
-        name: "Overall audio duration",
-        value: "Unknown",
-      },
-    ]);
+    this.stats
+      .show()
+      .pipe(takeUntil(this.unsubscribe))
+      .subscribe(({ summary, recent }) => {
+        updateValues(
+          "groupOne",
+          { index: 0, value: summary.projectsTotal },
+          { index: 1, value: summary.annotationsTotal },
+          { index: 2, value: summary.tagsTotal },
+          { index: 3, value: summary.sitesTotal },
+          { index: 4, value: summary.audioRecordingsTotal },
+          { index: 5, value: summary.usersTotal }
+        );
 
-    this.recentAnnotations = [{ updated: "Unknown" }];
+        updateValues(
+          "groupTwo",
+          { index: 0, value: summary.tagsAppliedUniqueTotal },
+          { index: 1, value: summary.tagsAppliedTotal },
+          { index: 2, value: recent.audioEventIds.size },
+          {
+            index: 3,
+            value: toRelative(summary.annotationsTotalDuration, { largest: 2 }),
+          },
+          { index: 4, value: summary.usersOnline },
+          {
+            index: 5,
+            value: fileSize(summary.audioRecordingsTotalSize, { round: 2 }),
+          },
+          { index: 6, value: summary.audioRecordingsRecent },
+          {
+            index: 7,
+            value: toRelative(summary.audioRecordingsTotalDuration, {
+              largest: 2,
+            }),
+          }
+        );
```

Comment:
ok this seems brittle. 

Can I suggest either:

- a arrow func on each definition such that with an updated value you can just `groupOne.map(x => x.new(updatedValue))`
- making the definitions funcs only (should be efficient since update only triggered by observable)
- or inverting the dictionary so that the `name` is the key and you can definitely assign in this step